jekyll/_cci2/test-splitting-tutorial.adoc

```diff
@@ -0,0 +1,248 @@
+---
+version:
+- Cloud
+- Server v3.x
+- Server v2.x
+---
+= Test splitting to speed up your pipelines
+:page-layout: classic-docs
+:page-liquid:
+:page-description: "Learn to split and run tests in parallel in CircleCI"
+:icons: font
+:toc: macro
+:toc-title:
+
+This guide walks you through a basic example of test splitting in CircleCI. 
```

Comment:
maybe expand to include why its useful, something like "This guide walks you through a basic example of using the CircleCI test splitting feature to speed up your pipelines." ?